tests/fixtures/optimizer/annotate_functions.sql

```diff
@@ -585,6 +585,14 @@ DOUBLE;
 DATETIME(2025, 1, 1, 12, 0, 0);
 DATETIME;
 
+# dialect: bigquery
+LAG(tbl.bigint_col, 1 , 2.5) OVER (ORDER BY tbl.bigint_col);
+DOUBLE;
```

Comment:
Are you sure that coercing into `DOUBLE` is correct here? Did you try this out in BigQuery? The docs say:

> Return data type: Same type as ``value_expression`.

Reply:
Sounds good